java/src/jmri/jmrit/logix/WarrantBundle.properties

```diff
@@ -94,11 +94,11 @@ speedKmph       = Kilometers per hour
 throttlesetting = Throttle setting
 badSpeed        = {0} is an invalid Throttle setting. Should be a decimal between 0.008 and 1.00
 badIncr			= {0} of {1} out of range for ramp increment.
-TimeIncrement   = Ramp Step Time (milliseconds)
-RampIncrement   = Ramp Step throttle increment
-timeWarning     = The time between speed changes must be at least 200 milliseconds.
-rampIncrWarning = {0} is an extreme amount for step increment. A value between .03 and .10 is more reasonable.
-ThrottleScale   = Throttle Setting/Speed Factor
+TimeIncrement   = Ramp Step Time:
+RampIncrement   = Ramp Step throttle increment:
+#timeToolTip     = The time between speed changes must be at least 200 milliseconds.
```

Comment:
The information in this commented out tooltip should be available to user in the ToolTipTimeIncrement tooltip. Otherwise, I am seeing no place where the user is presented with this constraint, since you removed the warning dialog.

Reply:
Will add it to the JPanel tooltip. In practice the value is bounded by the JSpinner model but I agree the user better be informed.